---
layout: page
title: Research overview
description: Some information about what we do
---

<section>

    <sub>May 2019 | Kate</sub>
    <hr/>

    <h4>Research overview</h4>

    <p>Antimicrobial resistance (AMR) is an urgent global public health crisis. Microbial genomics helps us understand the transmission of AMR among bacterial populations, and there is a need to harness this understanding to create interventions for public health. My group achieves this by studying Shigella pathogens using large-scale genomic epidemiology studies in high-income and low-to-middle-income countries (LMIC), laboratory experimentation and mathematical modelling.</p>

    <h5>DISEASE AND AMR BURDEN OF SHIGELLA: much disease, few management options</h5>

    <ul>
        <li>Top bacterial cause of moderate-severe diarrhoeal disease in children &lt;5 yrs in South Asia and Sub-saharan Africa<sup>1</sup></li>
        <li>Cause of foodborne illness and traveller’s diarrhoea in high-incomes nations</li>
        <li>Emerging globally as a sexually transmissible enteric illness (STEI) among men-who-have-sex-with-men (MSM)<sup>2</sup></li>
        <li>One of the WHO top dozen agents for which new antimicrobials are urgently needed<sup>3</sup></li>
        <li>No licensed vaccine for Shigella</li>
    </ul>

    <h5>SHIGELLA AS A MODEL: highly visible and traceable problem organism</h5>

    <p>Shigella are members of the Enterobacteriaceae, which includes 3 of the 12 WHO top-dozen AMR pathogens.3 Genomically, Shigella are highly changeable and elaborate AMR in ways relevant to all Enterobacteriaceae, including horizontal gene transfer and point mutations. Unlike many Enterobacteriaceae however, there is little evidence for a significant environmental or animal reservoir for Shigella, making them comparatively easy to track and, as they cause severe disease (dysentery), they are highly-visible in health care systems.</p>

    <h5>TRACKING AMR Shigella AS AN STEI OF MSM</h5>

    <p>The occurrence of highly AMR Shigella among MSM:</p>
    <ul>
        <li>is a public health issue</li>
        <li>represents a broader crisis of highly AMR-STIs in MSM, including gonorrhoea</li>
        <li>facilitates the study of Shigella AMR genomics in high-income nation healthcare surveillance in a dense transmission network with high antimicrobial usage</li>
    </ul>

    <p>In a large international collaboration, we identified a Shigella sublineage that is emerging globally (UK, France, Canada, Australia) among MSM and is associated with transmissible AMR (plasmid-borne macrolide resistance) <sup>2</sup>. This work was then built on to identify that the resistance plasmid was moving among different sublineages of Shigella transmitting among MSM. Plasmid acquisition <a href="https://www.youtube.com/watch?v=KiBgsnYoJac">drove epidemics of new Shigella sublineages</a><sup>4</sup>.</p>

    <p>We are currently extending this work in collaboration with PHE by:</p>

    <p>1) correlating the transmission behaviour of the plasmid in the laboratory to the public-health level incidence data of different Shigella sublineages and develop models to determine the impact of antimicrobial management strategies to inform stewardship</p>

    <p>2) determine whether the microbiota played a role in AMR transmission among MSM-associated Shigella strains (using mathematical modelling of epidemiological and genomic data in collaboration with the Liverpool Centre for Mathematics in Healthcare)</p>

    <p>We are also aiming to expand this work through local collaboration with Liverpool Royal and Broadgreen Hospitals Trust to study the burden of AMR in Enterobacteriaceae among MSM (using genomics and metagenomics), and continue to work with Public Health England’s National Reference laboratory on MSM-associated Shigella.</p>

    <h5>INFORMING MANAGEMENT OF SHIGELLOSIS IN RESOURCE-POOR REGIONS</h5>

    <p>MSM-associated Shigella is important and the genomic insights are relevant to less observable disease scenarios, including in LMICs where the greatest disease burden occurs (and contributes to broader AMR through travel). Infectious disease studies in LMIC are often complicated by a lack of systematic surveillance. We have previously engaged with existing surveillance networks (such as the foodborne illness transmission network PulseNet Latin America) to create a genomic benchmark for <a href="https://microreact.org/project/Shigella_sonnei_in_Latin_America">new pathogen databases</a><sup>5</sup>.</p>

    <p>To continue our work to address the Shigella disease burden in LMIC, we are studying the systematically-collected Shigella isolates from GEMS, a five-year aetiological study on moderate to severe diarrhoeal disease in children &lt;5 years conducted across seven nations in South Asia and SubSaharan Africa <sup>1</sup>. The GEMS work is in collaboration with Sharon Tennant at the University of Maryland, Baltimore and Neil Hall at the Earlham Institute in Norwich. We are also studying the genomic epidemiology of Shigella in South Africa in collaboration with the National Centre for Infectious Diseases in South Africa. By studying the genome sequences, we will:</p>

    <ul>
        <li>identify Shigella sublineages most responsible for disease in this key demographic</li>
        <li>evaluate local and regional antimicrobial treatment guidelines against pathogen AMR signatures (genotypic and phenotypic AMR correlate well in Shigella)</li>
        <li>work to aid the use of vaccines for Shigella in these regions including: identify relevant variation in existing vaccine candidates; better-characterise phenomena that contribute to vaccine escape (serotype switching); and identify novel vaccine candidates</li>
    </ul>

    <h5>STUDYING GENOMES OVER TIME</h5>

    <p>Underpinning our understanding the genomic epidemiology of pathogens is a fundamental understanding of their evolution. Historical isolates of Shigella show targeted gain of AMR and virulence characteristics over time (e.g. this <a href="https://www.youtube.com/watch?v=u-fbd9JpiMs">WW1 isolate study</a>)<sup>6</sup>. We are now seeking other genes with similar temporal signatures to AMR to determine what else contributes to pathogen persistence in populations over time. This work is in collaboration with the National Collection of Type Cultures and Tim Blower at the University of Durham </p>

    <h5>References</h5>

    <ol>
        <li>Kotloff et al (2013) Burden and aetiology of diarrhoeal disease in infants and young children in developing countries (the Global Enteric Multicenter Study, GEMS): a prospective, case-control study. The Lancet.</li>
        <li>Baker et al (2015) Intercontinental dissemination of azithromycin-resistant shigellosis through sexual transmission: a cross-sectional study. Lancet Infectious Diseases.</li>
        <li>World Health Organisation (2018) <a href="https://www.who.int/news-room/detail/27-02-2017-who-publishes-list-of-bacteria-for-which-new-antibiotics-are-urgently-needed">link</a></li>
        <li>Baker et al (2018) Horizontal antimicrobial resistance transfer drives epidemics of multiple Shigella species. Nature Communications.</li>
        <li>Baker et al (2016) Whole genome sequencing of Shigella sonnei through PulseNet Latin America and Caribbean: Advancing global surveillance of foodborne illnesses. Clinical Microbiology and Infection.</li>
        <li>Baker et al (2014) The extant WWI dysentery bacillus NCTC1: a genomic analysis. The Lancet.</li>
    </ol>
</section>
